tests/fixtures/atmos.yaml

```
@@ -0,0 +1,34 @@
+base_path: "./"
+
+components:
+  terraform:
+    # Set base path to the shared components directory
+    base_path: "components/terraform"
```

https://github.com/cloudposse/atmos/blob/main/tests/fixtures/scenarios/atmos-functions/atmos.yaml#L5
```suggestion
    base_path: "../../components/terraform"
```